SofaKernel/modules/SofaDeformable/MeshSpringForceField.inl

```diff
@@ -66,8 +66,8 @@ void MeshSpringForceField<DataTypes>::addSpring(std::set<std::pair<int,int> >& s
         sset.insert(std::make_pair(m2,m1));
     }
     Real l = ((this->mstate2->read(core::ConstVecCoordId::restPosition())->getValue())[m2] - (this->mstate1->read(core::ConstVecCoordId::restPosition())->getValue())[m1]).norm();
-     this->springs.beginEdit()->push_back(typename SpringForceField<DataTypes>::Spring(m1,m2,stiffness/l, damping/l, l, noCompression.getValue()));
-      this->springs.endEdit();
+    this->springs.beginEdit()->push_back(typename SpringForceField<DataTypes>::Spring(m1,m2,stiffness/l, damping/l, l, noCompression.getValue()));
```

Comment:
It is out of the topic of the PR but it would be nice to remove all these 'this'.

Reply:
See https://github.com/sofa-framework/sofa/pull/782/commits/3590a55b2e7ca5988501c353bad1ecc2264e1a0c